server/proxy/notify.go

```diff
@@ -74,7 +74,7 @@ func (p *Proxy) notifyUserChannel(channelID, userID string, joined bool, method
 
 	// If the user is a bot, process SubjectBot...Channel; only notify the
 	// app with the matching BotUserID.
-	allApps := p.store.App.AsMap()
+	allApps := p.store.App.ListAsMap()
```

Comment:
This is a behaviour change, and a wanted one I think. @levb Can you please double check?